awxkit/setup.py

```diff
@@ -67,7 +67,7 @@ def run(self):
     },
     include_package_data=True,
     install_requires=requirements,
-    python_requires=">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*",
+    python_requires=">=3.6",
```

Comment:
🎉

Reply:
Should we start adding Trove classifiers to the setup.py?  Are we intending to make awxkit, or at least awx-cli, available on pypi.org?

Reply:
Should we start adding Trove classifiers to the setup.py?  Are we intending to make awxkit, or at least awx-cli, available on pypi.org?

Reply:
We don't currently publish to PyPI, but if we did in the future then we should definitely add Trove classifiers 👍